### DISCUSSION TITLE
NT 2023.001

### DISCUSSION THREAD

**Discussion by giovaneboeing** [wmixvideo/nfe#883] [Geral]
Bom dia a todos, gostaria de saber se está sendo implementado na lib as alterações referentes a Nota Técnica 2023.001 publicada no dia 16/02/2023.

https://www.nfe.fazenda.gov.br/portal/exibirArquivo.aspx?conteudo=Li420kNzaH0=

**Comment by oliveiracontrolgas** [comment]
Opa, alguém está fazendo essas alterações?
